server/src/test-utils/environments/mock-entity-mikro-environment.ts

```diff
@@ -0,0 +1,44 @@
+import { mocks } from '..'
+import { ClubEntity } from '../../shared/infra/db/entities/club.entity'
+import { EventEntity } from '../../shared/infra/db/entities/event.entity'
+import { TagEntity } from '../../shared/infra/db/entities/tags/tag.entity'
+import { MikroEnvironmentVariables, MikroTestEnvironment } from './mikro-environment'
+
+export class MockEntityMikroTestEnvironment extends MikroTestEnvironment {
```

Comment:
What other test environment subclasses could we potentially have in the future?

Reply:
One example is if we want to set up only users, then all the events and clubs don't make much sense.